dex/models/_projects/zeroex/ethereum/zeroex_ethereum_native_fills.sql

```diff
@@ -256,25 +256,13 @@ WITH
       FROM {{ source('zeroex_ethereum', 'ExchangeProxy_evt_RfqOrderFilled') }} fills
       LEFT JOIN {{ source('prices', 'usd') }} tp ON
           date_trunc('minute', evt_block_time) = tp.minute
-          {% if is_incremental() %}
-          AND {{ incremental_predicate('tp.minute') }}
-          {% endif %}
-          {% if not is_incremental() %}
-          AND evt_block_time >= TIMESTAMP '{{zeroex_v3_start_date}}'
-          {% endif %}
```

Comment:
The incremental filters are removed for prices join, but this change may have minor impact.